src/jit/compiler.hpp

```diff
@@ -3127,6 +3127,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+//------------------------------------------------------------------------
+// compUpdateLife: Update the GC's masks, register's masks and reports change on variable's homes given a set of
+//    current live variables if changes have happened since "compCurLife".
+//
+// Arguments:
+//    newLife - the set of variables that are alive.
+//
+// Assumptions:
+//    The set of live variables reflects the result of only emitted code, it should not be considering the becoming
```

Comment:
grammar suggestion =>
`The set of live variables reflects the result of already emitted code. It should not consider live/dead instructions
becoming that have not been emitted yet. This is requires by "compChangeLife".`

Reply:
What do you think of changing your "consider live/dead instructions becoming" for "incoming live/dead instructions"?

Reply:
"compChangeLife" is the name of the function that has this assumption. Did you mean to write that name?